browser/ui/views/brave_help_bubble/brave_help_bubble_host_view.cc

```diff
@@ -25,7 +25,6 @@ namespace {
 constexpr int kWidth = 60;
 constexpr int kHeight = 60;
 constexpr base::TimeDelta kPulsingDuration = base::Milliseconds(1000);
-constexpr SkColor kBlockColor = SkColorSetRGB(52, 172, 224);
 constexpr SkPoint kPts[] = {{0, 0}, {60, 60}};
```

Comment:
Should I give array count here? even though the compiler will determine it for me.

@sangwoo108 @petemill

Reply:
I believe it's okay as is. Also, omitting it would make code more robust to changes on elements 😀